src/components/Characters/Robot/Chest.js

```diff
@@ -0,0 +1,29 @@
+        fill="#222"
+      />
+    </svg>
+  );
+}
+
+const Chest = styled(ChestArtwork)`
+  position: absolute;
+  top: 6.4rem;
+  left: 1.1rem;
+  z-index: ${zIndexArtworkPart + 3};
```

Comment:
I'm always curious about those _magic numbers_ 🤔why `3` not `2` or `5`?  😎

Reply:
I think it would be slight overkill to define separate const for each part. It's just z-index, pretty harmless stuff.